### PR TITLE
[admin_handler] create gflag for allowing global seqno

### PR DESCRIPTION
**Pull request by cpd85**

### CODE REVIEW

**Comment by cpd85** [comment]
the default case (false) is more common

File: rocksdb_admin/admin_handler.cpp [hunk 1]
```diff
@@ -53,6 +53,9 @@ DEFINE_int32(port, 9090, "Port of the server");
 DEFINE_string(shard_config_path, "",
              "Local path of file storing shard mapping for Aperture");
 
+DEFINE_bool(allow_global_seqno, false,
```
Comment (newpoo) [inline rocksdb_admin/admin_handler.cpp]: nit: rename it to rocksdb_allow_global_seqno, and see if we can put them on the same line?

File: rocksdb_admin/admin_handler.cpp [hunk 2]
```diff
@@ -722,7 +725,7 @@ void AdminHandler::async_tm_addS3SstFilesToDB(
   rocksdb::IngestExternalFileOptions ifo;
   ifo.move_files = true;
   /* allow for overlapping keys */
```
Comment (newpoo) [inline rocksdb_admin/admin_handler.cpp]: remove this comment now?

Comment (cpd85) [reply rocksdb_admin/admin_handler.cpp]: isn't it useful to say what FLAGS_ rocksdb_allow_global_seqno does?

**Comment by newpoo** [comment]
sure, I was suggesting that the comment is inaccurate now because allowing overlapping keys is now determined by a gflag.

**Pull request merged**